components/infobox/commons/infobox_game.lua

```diff
@@ -65,11 +112,53 @@ function Game:createInfobox()
 		Center{content = {args.footnotes}},
 	}
 
-	if Namespace.isMain() then
-		infobox:categories('Games')
+	-- Store LPDB data and Wiki-variables
+	if self:shouldStore(args) then
+		self:_setLpdbData(args, links)
 	end
 
 	return infobox:widgetInjector(self:createWidgetInjector()):build(widgets)
 end
 
+---@param args table
+---@param links table
+function Game:_setLpdbData(args, links)
+
+	local lpdbData = {
+		name = args.romanized_name or self.name,
+		image = args.image,
+		imagedark = args.imagedark,
+		date = args.releasedate,
+		type = 'game',
+		extradata = {
+			developer = table.concat(self:getAllArgsForBase(args, 'developer'), ';'),
+			publisher = table.concat(self:getAllArgsForBase(args, 'publisher'), ';'),
+			platform = table.concat(self:getAllArgsForBase(args, 'platform'), ';'),
```

Comment:
just as an fyi this will be bad to query

Reply:
What is the better way to query?

Reply:
well, if you want to query on them you would need to add each as sep key-value-pairs inside extradata